orchestration/src/main/java/com/sap/ai/sdk/orchestration/ContentFilter.java

```diff
@@ -0,0 +1,3 @@
+package com.sap.ai.sdk.orchestration;
+
+public sealed interface ContentFilter permits AzureContentFilter {}
```

Comment:
Sealed is quite useful here, as this is a marker interface, and users can't supply their own implementations, because we wouldn't have the implementation to translate those to DTOs.

With a higher Java version, we could even switch over implementations of the interface. That has the nice benefit that the compiler can force us to handle all implementations. But only with Java 21 🤷🏻